css/styles.css

```diff
@@ -0,0 +1,399 @@
+/* font pack*/
+
+    @import url('https://fonts.googleapis.com/css2?family=Bakbak+One&family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap');
+
+    @import url('https://fonts.googleapis.com/css2?family=Bakbak+One&family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+    @import url('https://fonts.googleapis.com/css2?family=Bakbak+One&family=Inter:wght@100..900&family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+/****************************************************************/
+
+html {
+    background-color: #0D0D11;
+}
+
+
+h1,h2,h3 {    
+    font-family: Bakbak one, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    color: #fff;
+}
+
+
+p {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    color: #FFFFFF;
+}
+
+body {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+.hidden {
+    display: none;
+}
+
+/*colored text in the headers*/
+
+
+.text-label, .colored-text {
+    color: #21e786;
+}
+
+/*header*/
+
+.main-header {
+    background-color:#040b11;
+        width: 1920px;
+        height: 108px;
+    }
+
+
+.header-nav {
+    font-size: 16px;
+    line-height: 150%;
+    letter-spacing: 0.1em;
+    text-transform: uppercase;
+    color: #fff;
+    display: inline-block;
+
+}
+
+/*links*/
+
+.nav-link-hdr a:link {
+    font-family: Bakbak One, sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 150%;
+    letter-spacing: 0.1em;
+    text-transform: uppercase;
+    color: #fff;
+}
+
+
+.nav-link-hdr a:hover {
+    color:#21e786;
+    text-decoration: underline ;
+}
+
+
+.nav-link-hdr a:visited {
+    color:#ffffffa7;
+}
+
+/*dropdown*/
+
+.header-nav-list {
+    justify-content: flex;
+    flex-direction: row;
+    gap: 32px;
+    list-style: none;
+}
+
+.nav-link-hdr:hover > .subnav {
+    display: block;
+}
+
+.subnav {
+    display: none;
+    min-width: 100%;
+    margin: 0;
+    padding: 0;
+    list-style: none; 
+}
+
+.nav-link-hdr {
+    float: left;
+    list-style:none
+}
+
+.hdr>div { 
+    display: inline-block;
+}
+
+.header-connect-btn {
+    font-family: Bakbak One, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+/*********************************************************/
+
+.tagline    {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding: 0px;
+}
+
+.main-tagline   {
+    width: 750px;
+    height: 240px;
```

Comment:
It's really cool that I tried to set the parameters for the block! I now recommend removing height everywhere. Because this is a very insidious property, it can break our site. Try to avoid setting height unnecessarily in the future. Yes, it's hard to understand now, but it will come with experience. The most important thing now is that the height should be according to the content, that is, the browser itself will decide how much it needs to occupy